scripts/managed-services/get-kafka-instance-state/get-kafka-instance-state.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+function main() {
+  local kafkaId=$1
```

Comment:
Would it be more user friendly to base things on cluster name?

Maybe use something like 

```bash
local clusterName=$1
local kafkaId=$(oc get pods -A -l strimzi.io/cluster=${clusterName} -o json \
    | jq -r ".items[].metadata.namespace" \
    | head -n1)
```

Which would give:
```bash
❯ echo $clusterName
sbarker-kafka
❯ kafkaId=$(oc get pods -A -l strimzi.io/cluster=${clusterName} -o json | \
 jq -r ".items[].metadata.namespace"  | \
 head -n1)
❯ echo $kafkaId
kafka-c6ens493tq20ii2uemig
```

Which is the full name of the namespace so need for `kafka-$kafkaId` usage.
 
Assuming `jq` is available in the backplane image.

Reply:
Jq is available

Reply:
As far as I'm aware, the CEE engineers will be getting the Kafka instance from something like the `MK - ID Finder dashboard`. @StevenTobin is that correct?

Reply:
Kafka cluster names are not unique so the id is the reliable way to identify which kafka cluster we want to target.